matrix_client/api.py

```diff
@@ -938,6 +938,17 @@ def send_to_device(self, event_type, messages, txn_id=None):
             content={"messages": messages}
         )
 
+    def get_encryption_algorithm(self, room_id):
```

Comment:
Can this be a generic `get_state_event` method instead? where URL is just built from `event_type`?